### ISSUE TITLE
Not able to run on MacOS Mavericks

### ISSUE THREAD

**Issue by loretoparisi** [cyrus-and/gdb-dashboard#10]
I've installed gdb through homebrew and gdb-dashboard as described with wget, but when I run gdb against a python script the gdb shell starts. What I'm missing?

**Comment by cyrus-and** [comment]
Did the prompt change to `>>>`? If so, everything is OK. The dashboard will show up once you load and run a program.

**Comment by cyrus-and** [comment]
> when I run gdb against a python script

BTW, can you please clarify what do you mean with that?

**Comment by loretoparisi** [comment]
Hello, so let me try again as you suggested. Supposed to have a simple test.py script, which is the sequence of commands, maybe I'm missing something.
I'm running

```Shell
macbookproloreto:lyriXmatch4iphone admin$ gdb --version
GNU gdb (GDB) 7.10
```

**Comment by cyrus-and** [comment]
I'm afraid I'm not fully understanding your problem so forgive the broad answer.

First of all show me the content of that Python script and how you're executing GDB (along with the output you get).

If that script contains GDB-related Python code (e.g. `gdb.some_function()`) then that script must reside in `~/.gdbinit.d/` and it will be loaded automatically. If instead you wish to execute it one time only you can use the `-ix` or `-x` options:

    gdb -x test.py

Also note that you need a program to debug in order to see dashboard. This is something you can try to see if there are problems in displaying the dashboard:

    echo 'int main() {}' > file.c
    gcc file.c
    gdb -ex 'b main' -ex r a.out
